Tools/baselineDef.cc

```diff
@@ -958,16 +1154,40 @@ float BaselineVessel::coreMT2calc(const TLorentzVector & fatJet1LVec, const TLor
 void BaselineVessel::operator()(NTupleReader& tr_)
 {
   tr = &tr_;
+  UseLepCleanJets();
```

Comment:
Is this a temporary hack?

Reply:
yes! Once we moved to new ntuple, this come be removed.